devicetypes/smartthings/springs-window-fashions-shade.src/springs-window-fashions-shade.groovy

```diff
@@ -15,255 +15,269 @@ import groovy.json.JsonOutput
-    }
-
-    tiles(scale: 2) {
-        multiAttributeTile(name:"windowShade", type: "generic", width: 6, height: 4){
-            tileAttribute ("device.windowShade", key: "PRIMARY_CONTROL") {
-                attributeState "open", label:'${name}', action:"close", icon:"st.shades.shade-open", backgroundColor:"#79b821", nextState:"closing"
-                attributeState "closed", label:'${name}', action:"open", icon:"st.shades.shade-closed", backgroundColor:"#ffffff", nextState:"opening"
-                attributeState "partially open", label:'Open', action:"close", icon:"st.shades.shade-open", backgroundColor:"#79b821", nextState:"closing"
-                attributeState "opening", label:'${name}', action:"stop", icon:"st.shades.shade-opening", backgroundColor:"#79b821", nextState:"partially open"
-                attributeState "closing", label:'${name}', action:"stop", icon:"st.shades.shade-closing", backgroundColor:"#ffffff", nextState:"partially open"
-            }
-            tileAttribute ("device.level", key: "SLIDER_CONTROL") {
-                attributeState "level", action:"setLevel"
-            }
-        }
-
-        standardTile("home", "device.level", width: 2, height: 2, decoration: "flat") {
-            state "default", label: "home", action:"presetPosition", icon:"st.Home.home2"
-        }
-
-        standardTile("refresh", "device.refresh", width: 2, height: 2, inactiveLabel: false, decoration: "flat") {
-            state "default", label:'', action:"refresh.refresh", icon:"st.secondary.refresh", nextState: "disabled"
-            state "disabled", label:'', action:"", icon:"st.secondary.refresh"
-        }
-
-        valueTile("battery", "device.battery", decoration: "flat", inactiveLabel: false, width: 2, height: 2) {
-            state "battery", label:'batt.', unit:"",
-                    backgroundColors:[
-                            [value: 0, color: "#bc2323"],
-                            [value: 6, color: "#44b621"]
-                    ]
-        }
-
-        preferences {
-            input "switchDirection", "bool", title: "Flip the orientation of the shade", defaultValue: false, required: false, displayDuringSetup: false
-//            input "preset", "number", title: "Default half-open position (1-100). Springs Window Fashions users should consult their manuals.", defaultValue: 50, required: false, displayDuringSetup: false
-        }
-
-        main(["windowShade"])
-        details(["windowShade", "home", "refresh", "battery"])
-
-    }
+	definition (name: "Springs Window Fashions Shade", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.blind") {
+		capability "Window Shade"
+		capability "Window Shade Level"
+		capability "Window Shade Preset"
+		capability "Battery"
+		capability "Refresh"
+		capability "Health Check"
+		capability "Actuator"
+		capability "Sensor"
+
+		command "stop"
+
+		capability "Switch Level"   // Leaving for second phase for Window Shade Level rollout
```

Comment:
I don't think we can actually remove this capability ever so I'd just move it up to the main list of capabilities